sdk/storage/azblob/appendblob/client_test.go

```diff
@@ -0,0 +1,1443 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package appendblob_test
+
+import (
+	"bytes"
+	"context"
+	"crypto/md5"
+	"strings"
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/streaming"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob/appendblob"
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob/blob"
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob/bloberror"
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob/container"
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob/internal/testcommon"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+func Test(t *testing.T) {
+	suite.Run(t, &AppendBlobRecordedTestsSuite{})
+	//suite.Run(t, &AppendBlobUnrecordedTestsSuite{})
```

Comment:
Should this be uncommented?

Reply:
No.  These tests don't have recordings at present.

Reply:
@mohsha-msft is it just that these tests are missing recordings, or is there more to it than that.

Reply:
Opened https://github.com/Azure/azure-sdk-for-go/issues/18935 to track this, we don't have to solve it here.